src/core/Akka.Tests/Loggers/CustomLogFormatterSpec.cs

```diff
@@ -0,0 +1,78 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="CustomLogFormatterSpec.cs" company="Akka.NET Project">
+//     Copyright (C) 2009-2023 Lightbend Inc. <http://www.lightbend.com>
+//     Copyright (C) 2013-2023 .NET Foundation <https://github.com/akkadotnet/akka.net>
+// </copyright>
+//-----------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Akka.Configuration;
+using Akka.Event;
+using Akka.TestKit;
+using FluentAssertions;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Akka.Tests.Loggers
+{
+    public class CustomLogFormatterSpec : AkkaSpec
+    {
+        // <CustomLogFormatter>
+        private class CustomLogFormatter : ILogMessageFormatter
+        {
+            public string Format(string format, params object[] args)
+            {
+                return string.Format("Custom: " + format, args);
+            }
+
+            public string Format(string format, IEnumerable<object> args)
+            {
+                return string.Format("Custom: " + format, args.ToArray());
+            }
+        }
+        // </CustomLogFormatter>
+
+        // <CustomLogFormatterConfig>
+        public static readonly Config Configuration = "akka.logger-formatter = \"Akka.Tests.Loggers.CustomLogFormatterSpec+CustomLogFormatter, Akka.Tests\"";
+        // </CustomLogFormatterConfig>
+        
+        public CustomLogFormatterSpec(ITestOutputHelper output) : base(Configuration, output)
+        {
+        }
+        
+        [Fact]
+        public async Task ShouldUseValidCustomLogFormatter()
```

Comment:
Validates that custom `ILogMessageFormatter` output actually makes it into the output events